profile/authCallback.php

```diff
@@ -25,15 +28,15 @@
     $status = $oidc->authenticate();
   }
   catch (Exception $ex){
-    $_SESSION['last_message'] = 'Caught exception: ' . $ex->getMessage() . ' <ERR/>';
-    header('Location:' . $CLIENT_ROOT . '/profile/index.php');
+    $_SESSION['last_message'] = $LANG['CAUGHT_EXCEPTION'] . $ex->getMessage() . ' <ERR/>';
+    header($LANG['LOCATION'] . $CLIENT_ROOT . '/profile/index.php');
```

Comment:
@Atticus29 , I think this doesn't need a lang tag, correct? Ed always removes them from my internationalizations, but I've never asked

Reply:
Correct. No lang tag needed here, either.